### DISCUSSION TITLE
Ability to add custom cli flags

### DISCUSSION THREAD

**Discussion by michaelschapira** [cockpit-project/cockpit-machines#1829] [Ideas]
I want to experiment with CoreOS. To boot / install and ignition file needs to be provided.
As per [documentation](https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-libvirt/#_launching_a_vm_instance)  several CLI parameters need to be provided :
`--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}"`

Thanks.

**Comment by garrett** [comment]
I've created an issue based on this @ https://github.com/cockpit-project/cockpit-machines/issues/1835 as I think we need it to properly support creating CoreOS VMs.
